### ISSUE TITLE
Feature request: Feedback image and generic text 

### ISSUE THREAD

**Issue by guywillis** [adaptlearning/adapt-contrib-core#222]
Feature request to support images, with alt text if required, in question feedback. Images should be able to mirror the question feedback text so there are correct, partly correct not / final, and incorrect not final versions.

If an image is defined, the layout of the text and image is split 60% / 40% in favour of the text. If no image is defined the text defaults to 100% width.

Allow generic feedback title for all forms of feedback with ability to override title per feedback state.

Schemas will also need to be updated.

Suggestion:

```
"_feedback": {
  "_title": "Generic title",
  "_correct": {
    "_title": "Optional, override, correct title",
    "body": "Correct feedback text",
    "_graphic": {
      "alt": "Alt text",
      "_src": "course/en/images/correct.png",
    }
  },
  "_partlyCorrect": {
    "_notFinal": {
      "_title": "Optional, override, partly correct not final title",
      "body": "Partly correct not final feedback text",
      "_graphic": {
        "alt": "Alt text",
        "_src": "course/en/images/partly-correct-not-final.png"
      }
    },
    "_final": {
      "_title": "Optional, override, partly correct final title",
      "body": "Partly correct final feedback text",
      "_graphic": {
        "alt": "Alt text",
        "_src": "course/en/images/partly-correct-final.png"
      }
    }
  },
  "_incorrect": {
    "_notFinal": {
      "_title": "Optional, override, incorrect not final title",
      "body": "Incorrect not final feedback text",
      "_graphic": {
        "alt": "Alt text",
        "_src": "course/en/images/incorrect-not-final.png"
      }
    },
    "_final": {
      "_title": "Optional, override, incorrect final title",
      "body": "Incorrect final feedback text",
      "_graphic": {
        "alt": "Alt text",
        "_src": "course/en/images/incorrect-final.png"
      }
    }
  }
}
```
